hathor/event/event_manager.py

```diff
@@ -153,6 +153,15 @@ def load_finished(self):
         )
         self._event_storage.save_node_state(NodeState.SYNC)
 
+    def full_node_crashed(self) -> None:
```

Comment:
Maybe rename to `on_full_node_crash()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/809/commits/2938c0855a3efcceece808b8116061e25489dce9

Reply:
Thanks!